### ISSUE TITLE
Extended check: checking for Elixir.IO.inspect calls in addition to IO.inspect calls

### ISSUE THREAD

**Issue by juhalehtonen** [rrrene/credo-proposals#72]
#### TL/DR

Check for Elixir.IO.inspect in addition to IO.inspect

#### With background

We at work have been using Credo extensively, and are extremely happy with it (thank you to all the contributors!). We typically run Credo as part of a pipeline for merge requests.

Recently, we ran into a situation where a merge request was OK'd by our Credo step, even though we had a check enabled for having no stray IO.inspect calls (`Credo.Check.Warning.IoInspect`). Soon we realized that this was because we were using a call to `Elixir.IO.inspect`, which is technically the exact same module and function as just `IO.inspect`. Our reason for doing this was due to having a collection of modules that end up being aliased as `IO`, which conflicts with the namespace.

I already did this change locally, but I wanted to check if it was worthwhile to send a PR to add this to Credo core. So the question is: does adding this for Credo make sense, or shall we simply keep the change in our private custom checks? I recognize that the use case might be somewhat niche, even if technically correct.

### Which existing behaviour would change?

Credo.Check.Warning.IoInspect would be extended to also cover `Elixir.IO.inspect` calls.

**Comment by rrrene** [comment]
I would merge a PR adding this :+1:

**Comment by rrrene** [comment]
Closing as this is part of Credo. 🎉
